env.ts

```diff
@@ -0,0 +1,30 @@
+import { resolve } from "node:path";
+import { fileURLToPath } from "node:url";
+
+import { type } from "arktype";
+import { config } from "dotenv";
+
+const __dirname = fileURLToPath(new URL(".", import.meta.url));
+
+/**
+ * ESbuild will pick up on this and copy the env file to the output folder.
+ */
+// eslint-disable-next-line
+try {
+  require("./.env");
+} catch {}
```

Comment:
```suggestion
/**
 * esbuild will pick up on this and copy the env file to the output folder.
 */
try {
  require("./.env");
} catch { /* noop */ }
```
This actually works and is IMO more concise than using eslint-disable macros.